src/Avalonia.Controls/DateTimePickers/TimePickerPresenter.cs

```diff
@@ -28,7 +28,9 @@ namespace Avalonia.Controls
     [TemplatePart("PART_PeriodSelector",   typeof(DateTimePickerPanel), IsRequired = true)]
     [TemplatePart("PART_PeriodUpButton",   typeof(RepeatButton))]
     [TemplatePart("PART_PickerContainer",  typeof(Grid), IsRequired = true)]
-    [TemplatePart("PART_ThirdSpacer",     typeof(Rectangle), IsRequired = true)]
+    [TemplatePart("PART_FirstSpacer",      typeof(Rectangle), IsRequired = true)]
+    [TemplatePart("PART_SecondSpacer",     typeof(Rectangle), IsRequired = true)]
+    [TemplatePart("PART_ThirdSpacer",      typeof(Rectangle), IsRequired = true)]
```

Comment:
Just a sanity check, are these really required? I.e. will control work fine, if these spacers were missing, without any C# code change.

Reply:
`ApplyTemplate` crashed when I didn't include them in the test template, so I added them here as they seemed to be missing.

Reply:
Looks like FirstSpacer doesn't need to exist (isn't used at all, which is a bit odd), but SecondSpacer is assumed in the code. I'll remove FirstSpacer or make any other suggested changes tomorrow.

Reply:
Yep, they're unfortunately needed currently. We should refactor these templates someday: I'm sure a lot of parts could be made optional and/or simple styles added to the themes instead. Anyways, such a refactoring isn't the point of this PR :)